.github/workflows/ci.yml

```diff
@@ -11,9 +11,11 @@ jobs:
     name: Check the dist/ folder is up to date
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v3
-      - run: npm install --no-save
-      - run: npm run all
+      - uses: actions/checkout@v4
+      - uses: DeterminateSystems/nix-installer-action@main
+      - uses: DeterminateSystems/magic-nix-cache-action@main
+      - run: nix develop --command pnpm install
```

Comment:
not sure if this applies to pnpm, but do you want/need some equivalent of `--no-save`? Seems like maybe that was just to avoid dirtying the tree mid-run?

Reply:
We're doing this deliberately actually, to make sure it is clean and not an outdated lockfile.